xtra-libp2p/src/endpoint.rs

```diff
@@ -267,13 +280,13 @@ impl Endpoint {
         self.notify_connection_dropped(*peer).await;
     }
 
-    #[instrument(skip(control), ret, err)]
+    #[instrument(skip(control, connection_timeout), ret(Display), err)]
```

Comment:
#2467 removes the ret - we can probably remove it here too

Reply:
Ok, I'll drop this commit and remove the ret

Reply:
Shall I close #2467 in favour of this?